spec.md

```diff
@@ -153,11 +171,14 @@ the expanded form.
 
 ## Version top-level element
 
-A top-level version property is required by the specification. Version MUST be 3.x or later, legacy docker-compose 1.x and 2.x are not included as part of this specification. Implementations MAY accept such legacy formats for compatibility purposes.
+Top-level `version` property is defined by the specification for backward compatibility but is only informative. 
 
-The specification format follows [Semantic Versioning](https://semver.org), which means that the file format is backward compatible within a major version set. As the specification evolves, minor versions MAY introduce new elements and MAY deprecate others for removal in the next major version.
+A Compose implementation SHOULD NOT use this version to select an exact schema to validate the Compose file, but 
+prefer the most recent schema at the time it has been designed. 
 
-Implementations MAY ignore attributes used in a configuration file that are not supported by the declared version, whenever they are valid for a more recent version. If they do, a warning message MUST inform the user.
+Compose implementations SHOULD validate they can fully parse the Compose file. If some fields are unknown, typically 
+because the Compose file was written with fields defined by a newer version of the specification, Compose implementations 
+SHOULD warn the user. Compose implementations MAY offer options to ignore unknown fields (as defined by ["loose"](#Requirements_and_optional_attributes) mode).
```

Comment:
GitHub uses dashes to separate words, so I believe the right link fragment would be `#requirements-and-optional-attributes`.